packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -828,23 +822,30 @@ export const Table = <
         data-iui-size={density === 'default' ? undefined : density}
         {...ariaDataAttributes}
       >
-        <div
-          className='iui-table-header-wrapper'
-          ref={headerRef}
-          onScroll={() => {
-            if (headerRef.current && bodyRef.current) {
-              bodyRef.current.scrollLeft = headerRef.current.scrollLeft;
-              updateStickyState();
-            }
-          }}
-        >
-          <div className='iui-table-header'>
-            {headerGroups.slice(1).map((headerGroup: HeaderGroup<T>) => {
-              const headerGroupProps = headerGroup.getHeaderGroupProps({
-                className: 'iui-table-row',
-              });
-              return (
-                <div {...headerGroupProps} key={headerGroupProps.key}>
+        {headerGroups.map((headerGroup: HeaderGroup<T>) => {
+          // There may be a better solution for this, but for now I'm filtering out the placeholder cells using header.id
+          headerGroup.headers = headerGroup.headers.filter(
+            (header) =>
+              !header.id.includes('iui-table-checkbox-selector_placeholder') &&
+              !header.id.includes('iui-table-expander_placeholder'),
+          );
+          const headerGroupProps = headerGroup.getHeaderGroupProps({
+            className: 'iui-table-row',
+          });
+          return (
+            <div
+              className='iui-table-header-wrapper'
```

Comment:
Shouldn't this `header-wrapper` wrap all groups?

Reply:
This is what Horiztonal Scroll looks like when the `Header` is specified now
![image](https://user-images.githubusercontent.com/78564221/201441357-0dd30c35-1aae-4e6e-a2a0-088b0043838a.png)

After wrapping all of the header groups in `iui-table-header-wrapper`, the result looks like this
![TableRowHeaderHorizontalScroll](https://user-images.githubusercontent.com/78564221/201441424-c3ed227f-4f3b-4956-92d3-8fbe5b4c0634.gif)

Reply:
i think this is a css bug. the wrapper needs flex-direction: column probably

Reply:
@adamhock Does adding `flex-direction` fix the problem with `header-wrapper` wrapping all groups?

Reply:
I don't think we are supporting columns grouping so there is no need to fix this.

Reply:
Adding `flex-direction` does solve this problem, but there's another problem that occurs after this change is made.

![TableRowHeaderHorizontalScroll](https://user-images.githubusercontent.com/78564221/201635290-4d25d60e-789e-40e3-ab7d-c65b8be6ea73.gif)

Reply:
> I don't think we are supporting columns grouping so there is no need to fix this.

Ok, so lets merge this as-is. But we need to clear in our messaging that we do not support grouped headers yet.